src/app/screens/gamepage/index.ts

```diff
@@ -179,6 +183,21 @@ export class GamePage {
     })
   }
 
+  private recruitUnits() {
+    let time = 2000
```

Comment:
it can't be set here, such things should belong to concrete unit class

Reply:
![image](https://user-images.githubusercontent.com/38506213/106730660-e6a3c580-660e-11eb-9299-596d1960c1e1.png)